client/reader/like-button/index.jsx

```diff
@@ -82,6 +89,12 @@ class ReaderLikeButton extends Component {
 						context={ this.likeButtonRef.current }
 					/>
 				) }
+				{ showJoinConversationModal && (
```

Comment:
nit: I don't think you need this check because the modal would be hidden anyway

Reply:
Yeah good point